### PR TITLE
curryst:0.1.0

### PR DESCRIPTION
**Pull request by pauladam94**
I am submitting
- [x] an update for a not accepted pull request (package inferrule)

Description: Easy to use inference rule package.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

### CODE REVIEW

**Comment by pauladam94** [comment]
This continue the Pull Request : https://github.com/typst/packages/pull/190#issue-1954840938

**Comment by pauladam94** [comment]
There was still the wrong name in the `typst.toml`file. I just fixed it to curryst.

File: packages/preview/curryst/0.1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Prooftree
```
Comment (Thumuss) [inline packages/preview/curryst/0.1.0/README.md:1]: The name isn't correct

Comment (pauladam94) [reply packages/preview/curryst/0.1.0/README.md]: Thank you ! This has been changed is the last commit.

**Comment by laurmaedje** [comment]
The LICENSE file is empty and is missing attribution of the original author. Independently of that, I think it would be better to get in touch with the original author about contributing the improvements instead of creating a fork.

**Comment by laurmaedje** [comment]
okay, makes sense!

**Comment by max397574** [comment]
just curious
what's the origin of this name?

**Comment by pauladam94** [comment]
Curry is a researcher in math and informatics that worked in his life about formal proof (where derivation tree are a very common).
Wikipedia of his famous theorem : [Curry Howard correspondence Wikipedia](https://en.wikipedia.org/wiki/Curry%E2%80%93Howard_correspondence).

**Comment by laurmaedje** [comment]
Thanks!

**Pull request merged**